### ISSUE TITLE
Using "dashboards" for extra ports doesn't appear to work.

### ISSUE THREAD

**Issue by GrahamDumpleton** [openshift-labs/learn-katacoda#533]
For the imageid of `openshift-3-11`, adding `dashboards` section doesn't work for anything except port 8443.

If use docker run to start image as:

```
docker run -p 8080:10080 quay.io/xxx/yyy
```

from the terminal, you can subsequently do:

```
curl master:8080
```

with no issues, and also:

```
curl -k https://master:8443
```

But you can't set up an entry in `dashboards` to use it.

```
    "backend": {
        "imageid": "openshift-3-11"
    },
    "environment": {
        "showdashboard": true,
        "dashboards": [
            {"name": "MyApp", "port": 8080}
        ],
        "uilayout": "terminal",
        "hideintro": true,
        "hidefinish": true
    },
```

Not sure if this is some hack due to fact that 8443 has to be `https`, so may be for this image it is treating all possible dashboard ports as `https` and so it fails since this is only `http`.

Tested running the same docker image under:

* https://katacoda.com/courses/jenkins/build-docker-images

as test, and for it you can get through the application running on port 8080 fine. So is specific to the OpenShift images.

Would really like to have this working.

**Comment by GrahamDumpleton** [comment]
FWIW, using:

```
        "dashboards": [
            {"name": "MyApp", "href": "http://[[HOST_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com"}
        ],
```

doesn't seem to help.

This `href` option isn't documented, but is used in example at:

* https://github.com/katacoda/scenario-examples/blob/33d11b27bc33598b4b51129bf3c67a0aa450a4aa/dashboard-tabs/index.json

**Comment by BenHall** [comment]
Hi @GrahamDumpleton 

Are you referring to a UI issue with the tabs not displaying, or that the link generated by the tab doesn't connect to port `8080`?

I believe you are referring to the latter. I've tested starting a container listening on a port on both 3.6 and 3.11 instances. The 3.6 works as expected.

With 3.11 we also see a error with us unable to reach the target port of 8080 or 8081 that was specified when we launched the container.

`curl: (7) Failed to connect to 172.17.0.15 port 8081: Connection timed out`

We are investigating and will update you.
